defs.mak

```diff
@@ -94,7 +94,7 @@ OEENCLAVE_LIBDIR = $(BUILDDIR)/openenclave/lib/openenclave/enclave
 
 OEENCLAVE_LDFLAGS = $(shell PKG_CONFIG_PATH=$(BUILDDIR)/openenclave/share/pkgconfig pkg-config oeenclave-gcc --libs)
 
-OEENCLAVE_LDFLAGS += $(shell PKG_CONFIG_PATH=$(BUILDDIR)/openenclave/share/pkgconfig pkg-config oeenclave-gcc --variable=mbedtlslibs)
+OEENCLAVE_LDFLAGS += $(shell PKG_CONFIG_PATH=$(BUILDDIR)/openenclave/share/pkgconfig pkg-config oeenclave-gcc --variable=${MYST_OE_CRYPTO}libs)
 
 OEENCLAVE_CFLAGS = $(shell PKG_CONFIG_PATH=$(BUILDDIR)/openenclave/share/pkgconfig pkg-config oeenclave-gcc --cflags) -g
```

Comment:
To build with OpenSSL 3.1, up above, we need to add:
```OEENCLAVE_INCLUDES += -I$(OE_INCDIR)/openenclave/3rdparty/openssl_3```
And this needs to be included _before_ the line:
```OEENCLAVE_INCLUDES += -I$(OE_INCDIR)/openenclave/3rdparty```

There's some information about this here:
https://github.com/openenclave/openenclave/blob/cdeb95c1ec163117de409295333b6b2702013e08/pkgconfig/README.md#building-enclave-applications
"OpenSSL 3.0 needs a different set of flags from cflags, because it uses additional include directories."